internal/distributed/rootcoord/service.go

```diff
@@ -258,6 +253,11 @@ func (s *Server) startGrpcLoop(port int) {
 func (s *Server) start() error {
 	log.Debug("RootCoord Core start ...")
 	if err := s.rootCoord.Start(); err != nil {
+		log.Error(err.Error())
```

Comment:
log format can be refine in the future

Reply:
Yeah, I will sort out the startup logic of each component, and sort out these logs by the way.